### PR TITLE
add COMR frame with test

### PR DESCRIPTION
**Pull request by m13m**

### CODE REVIEW

File: lib/id3vx/frame.ex [hunk 1]
```diff
@@ -496,6 +519,54 @@ defmodule Id3vx.Frame do
...
+      recieved_as: recieved_as,
+      seller_name: seller_name,
+      description: description,
+      picture_mime: picture_mime,
+      logo: logo
+    } = frame.data
+
+    null_byte = get_null_byte(encoding)
+    encoding_byte = get_encoding_byte(encoding)
+    contact_url = convert_string(:iso8859_1, contact_url)
```
Comment (lawik) [inline lib/id3vx/frame.ex:537]: This conversion should also be applied to:

- price
- valid until
- picture mime

File: lib/id3vx/frame.ex [hunk 2]
```diff
@@ -496,6 +519,54 @@ defmodule Id3vx.Frame do
...
+
+    recieved_as =
+      @recieved_as_type
+      |> Utils.flip_map()
+      |> Map.get(recieved_as)
+
+    IO.inspect(recieved_as)
```
Comment (lawik) [inline lib/id3vx/frame.ex:544]: Njet!
No inspects :)

**Review by lawik — CHANGES_REQUESTED** [review]
Looks good, just a few things.

**Comment by m13m** [comment]
ready for merge ✨

**Pull request merged**